script.js

```diff
@@ -318,7 +318,13 @@ function addEventListeners(
  *
  */
 function addUpStudentCredits(reportCardTableElement) {
-  // code goes here
+  let creditElems = reportCardTableElement.querySelectorAll(".credit");
+  let creditSum = 0;
+  for (let credit of creditElems) {
+    creditSum += parseInt(credit.innerHTML);
```

Comment:
One thing to consider here is what happens if parseInt returns `NaN`, which is possible for scenarios like `parseInt('a')`. Adding `NaN` to a number would result in `NaN` so `creditSum` would be invalid. You could use functions like `isNaN` to ensure that all creditSum values are numbers before getting the summation and anything that isn't would just be 0. This is very important with JavaScript because there is no type checking unless you use libraries like Flow to perform that type checking for you.